scripts/snyk-test.js

```diff
@@ -0,0 +1,89 @@
+const childProcess = require('child_process');
+const path = require('path');
+const { promises: fs } = require('fs');
+const os = require('os');
+const { glob } = require('glob');
+
+async function spawn(cmd, args, opts) {
+  return new Promise((resolve, reject) => {
+    const proc = childProcess.spawn(cmd, args, opts);
+    proc.on('error', (err) => reject(err));
+    proc.on('close', (code) => resolve(code));
+  });
+}
+
+async function snykTest(cwd) {
+  const tmpPath = path.join(os.tmpdir(), 'tempfile-' + Date.now());
+
+  try {
+    console.info(`testing ${cwd} ...`);
+    await fs.mkdir(path.join(cwd, `node_modules`), { recursive: true });
+
+    await spawn(
+      'npx',
+      [
+        'snyk',
+        'test',
+        '--all-projects',
+        '--severity-threshold=low',
+        '--dev',
+        `--json-file-output=${tmpPath}`,
+      ],
+      { cwd, stdio: 'inherit' }
+    );
+
+    const res = JSON.parse(await fs.readFile(tmpPath));
+    console.info(`testing ${cwd} done.`);
+    return res;
+  } catch (err) {
+    console.error(`testing ${cwd} failed. ${err.message}`);
+  } finally {
+    try {
+      await fs.rm(tmpPath);
```

Comment:
```suggestion
      await fs.rm(tmpPath, { recursive: true });
```

Reply:
does it have an effect if is always a file?

Reply:
ah no you’re right, I didn’t notice that this only a single file – sorry!